charts/trino/values.yaml

```
@@ -173,6 +173,31 @@ resourceGroups: {}
   #       ]
   #     }
 
+sessionProperty: {}
```

Let's rename it here and in the manifests where it's referenced.
```suggestion
sessionPropertyConfig: {}
```

---

I was following the same pattern as `resourceGroups`. If renamed as `sessionPropertyConfig`, in the manifest it will look like
 `{{- .Values.sessionPropertyConfig.sessionPropertyConfig | nindent 4 }}`
 Does it look redundant?

---

Right, it's a bit easier with resource groups since it's shorter. It's tricky here to keep it consistent with both other sections in the chart and Trino property file names. I'm ok with this duplication.